src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionCtx.java

```diff
@@ -162,12 +163,17 @@ public void assignParams(MultiMap params) throws IllegalArgumentException {
      * @see {@link com.google.common.hash.Hashing#sipHash24()}
      */
     private String createCacheKey(MultiMap params) {
-        StringBuilder sb = new StringBuilder()
-                .append(ImageRegionCtx.class.getName());
+        TreeSet<String> orderedKeys = new TreeSet<String>();
+        //Get an ordered set of keys
         for (Entry<String, String> entry : params) {
```

Comment:
I think we're probably okay to remove the debugging and swap this out for:

```
TreeSet<String> orderedKeys = new TreeSet<String>(params.names());
```

Assuming I'm reading the documentation of `MultiMap` correctly.